### ISSUE TITLE
Failded to list playlist 

### ISSUE THREAD

**Issue by guizhenwei** [Mic92/python-mpd2#88]
Hi, **MPD** just released the **latest** version(**0.20.10**) for **Windows** platform days ago,

You can find the latest binary file at:

https://www.musicpd.org/download/win32/ 
(Its actually Win64, not Win32.) 

After testing i found the below issue.

Environments:
```
MPD:  0.20.10    
OS:  Windows 10 Pro 64bit
Python:  2.7.13 64bit
python-mpd2:  0.5.5
```

In short, with the following code(related part only):

```
    handler = mpd.MPDClient(use_unicode=True)
    handler.connect(host, port)
    
    playlists= handler.listplaylists()
```

Problem below:

```
ERROR: Traceback (most recent call last):    

  File "C:\Python27\lib\site-packages\mpd.py", line 629, in decorator
    return wrapper(self, name, args, bound_decorator(self, returnValue))
  File "C:\Python27\lib\site-packages\mpd.py", line 254, in _execute
    return retval()
  File "C:\Python27\lib\site-packages\mpd.py", line 623, in decorator
    return function(self, *args, **kwargs)
  File "C:\Python27\lib\site-packages\mpd.py", line 431, in _fetch_playlists
    return self._fetch_objects(["playlist"])
  File "C:\Python27\lib\site-packages\mpd.py", line 410, in _fetch_objects
    return self._wrap_iterator(self._read_objects(delimiters))
  File "C:\Python27\lib\site-packages\mpd.py", line 374, in _wrap_iterator
    return list(iterator)
  File "C:\Python27\lib\site-packages\mpd.py", line 332, in _read_objects
    for key, value in self._read_pairs():
  File "C:\Python27\lib\site-packages\mpd.py", line 311, in _read_pairs
    pair = self._read_pair(separator)
  File "C:\Python27\lib\site-packages\mpd.py", line 302, in _read_pair
    line = self._read_line()
  File "C:\Python27\lib\site-packages\mpd.py", line 284, in _read_line
    line = decode_str(line)
  File "C:\Python27\lib\site-packages\mpd.py", line 33, in decode_str
    return s.decode("utf-8")
  File "C:\Python27\lib\encodings\utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)

UnicodeDecodeError: 'utf8' codec can't decode byte 0xd5 in position 36: invalid continuation byte

```

**Comment by Mic92** [comment]
Remind of a bug, we had in the past: https://github.com/Mic92/python-mpd2/issues/49#issue-52367182

**Comment by guizhenwei** [comment]
> Remind of a bug, we had in the past: #49 (comment)
> It was also on windows and do you have songs with Chinese characters?

Yes. Pretty much the same. I almost forget that i filed the issue before.
Yes, i have songs with Chinese characters.

Here i upload the playlist file for reference. (From .m3u renamed to .txt)

[360.txt](https://github.com/Mic92/python-mpd2/files/1254226/360.txt)

**Comment by guizhenwei** [comment]
I also reported the related issue to MPD issues list:
https://github.com/MusicPlayerDaemon/MPD/issues/102

**Comment by guizhenwei** [comment]
This issue has been fixed by upstream project MPD.
The latest released version(MPD 0.2.11 for win32) just fixed this issue.
